SimPEG/EM/Static/DC/IODC.py

```diff
@@ -0,0 +1,672 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib
+import properties
+import warnings
+
+import SimPEG
+from SimPEG import Utils, Mesh
+from . import SrcDC as Src
+from . import RxDC as Rx
+from .SurveyDC import Survey_ky, Survey
+
+
```

Comment:
import warnings
warnings.warn("code under construction - API might change in the future")